client/src/components/add-job-post-component/addJobPosting.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import dayjs from "dayjs";
+import AddJobLead from "./index";
+import UserType from "../../prop-types/UserType";
+
+function AddJobLeadParent({ currUser }) {
+  const initialState = {
+    jobInfo: {
+      id: 0,
+      title: "",
+      employer: "",
+      location: "",
+      minCompensation: null,
+      maxCompensation: null,
+      compensationRate: "",
+      hoursPerWeek: null,
+      creationDate: dayjs(),
+      expirationDate: dayjs().add(1, "month"),
+      employmentType: "",
+      additionalInfo: "",
```

Comment:
```suggestion
      description: "",
```
Can you make this change everywhere?